File: runelite/runelite-client/src/main/java/net/runelite/client/plugins/gamedataserver/model/config/LiveConfig.java
package net.runelite.client.plugins.gamedataserver.model.config;

import lombok.Builder;
import lombok.Data;

import java.util.List;

@Builder
@Data
public class LiveConfig {
	private boolean autotraderOn;
	private TopLevelConfig topLevelConfig;
	private List<StratConfig> stratConfigs;
}

File: autotrader/src/clients/gds/models/config/live_config.py
from dataclasses import dataclass
from typing import List

from clients.gds.models.config.strat_config import StratConfig
from clients.gds.models.config.top_level_config import TopLevelConfig


@dataclass
class LiveConfig:
    trading_enabled: bool
    top_level_config: TopLevelConfig
    strat_configs: List[StratConfig]
